### PR TITLE
feat: add multi-language support (Chinese, English and French)

### PR DESCRIPTION
**Pull request by kerms**
Used .resx resource file to achieve the localization function.
The default language is english, and auto-selected based on the system language (WPF built-in behavior).

### CODE REVIEW

**Comment by windowsair** [comment]
Amzing work! I had considered the feature of multi-language before, but I put it on hold due to time constraints and other reasons. Please give me some time to review your pr. 😀

**Comment by windowsair** [comment]
How about using i18n libraries like WPFLocalizeExtension? Would it be more flexible and convenient?

File: elaphureLink.Wpf/Properties/Resources.Designer.cs
Comment (windowsair) [inline elaphureLink.Wpf/Properties/Resources.Designer.cs:1]: ci build fail: 
```bash
CSC : error CS2001: Source file 'D:\a\elaphureLink\elaphureLink\elaphureLink.Wpf\Properties\Resources.Designer.cs' could not be found. 
```

Should we delete this?

Comment (windowsair) [reply elaphureLink.Wpf/Properties/Resources.Designer.cs:1]: Well, it looks like these are modifications to the source files. Do you need to submit the changes to csproj along with them?

Comment (kerms) [reply elaphureLink.Wpf/Properties/Resources.Designer.cs:1]: These files were empty(Resources.resx/Resources.resx.cs), I might delete them.

Comment (kerms) [reply elaphureLink.Wpf/Properties/Resources.Designer.cs:1]: > Well, it looks like these are modifications to the source files. Do you need to submit the changes to csproj along with them?

Oups, I didn't submit changes to .csproj, I'm beginner to C# and WPF, should I submit the .csproj file along with the commit ?

**Comment by kerms** [comment]
> How about using i18n libraries like WPFLocalizeExtension? Would it be more flexible and convenient?

😒I'm not expert in C#/WPF, I just used the first solution comes to me.

**Comment by windowsair** [comment]
> > How about using i18n libraries like WPFLocalizeExtension? Would it be more flexible and convenient?
> 
> 😒I'm not expert in C#/WPF, I just used the first solution comes to me.

The core ideas of the two method are consistent.  Maybe we can migrate to an i18n library. Now let's solve the compile issue first.  😉

File: elaphureLink.Wpf/Properties/Resources.Designer.cs
Comment (windowsair) [reply elaphureLink.Wpf/Properties/Resources.Designer.cs:1]: > > Well, it looks like these are modifications to the source files. Do you need to submit the changes to csproj along with them?
> 
> Oups, I didn't submit changes to .csproj, I'm beginner to C# and WPF, should I submit the .csproj file along with the commit ?

Yes. You can submit the changes to related files such as .csproj on the basis of this patch.

**Comment by kerms** [comment]
I

> > > How about using i18n libraries like WPFLocalizeExtension? Would it be more flexible and convenient?
> > 
> > 
> > 😒I'm not expert in C#/WPF, I just used the first solution comes to me.
> 
> The core ideas of the two method are consistent. Maybe we can migrate to an i18n library. Now let's solve the compile issue first. 😉

I will create a new PR tomorrow by cleaning all non-staged changes on my local repo. I close this PR for the moment.

**Comment by windowsair** [comment]
> I
> 
> > > > How about using i18n libraries like WPFLocalizeExtension? Would it be more flexible and convenient?
> > > 
> > > 
> > > 😒I'm not expert in C#/WPF, I just used the first solution comes to me.
> > 
> > 
> > The core ideas of the two method are consistent. Maybe we can migrate to an i18n library. Now let's solve the compile issue first. 😉
> 
> I will create a new PR tomorrow by cleaning all non-staged changes on my local repo. I close this PR for the moment.

Thank you for your commit!
You don't need to close the this pr, you can just submit a new commit on this pr and push it to github.  Your new modification will be visible on this page. 😉

**Pull request closed without merging**